yb-voyager/src/srcdb/postgres.go

```diff
@@ -180,3 +190,33 @@ func (pg *PostgreSQL) ExportDataPostProcessing(exportDir string, tablesProgressM
 	}
 	dfd.Save()
 }
+
+//find correct path of pg command depending on the version required
+func FindCorrectPGCommandPath(pathVariable *string, executableName string) {
+	paths, err := findAllExecutablesInPath(executableName)
+	if err != nil {
+		utils.ErrExit("error in finding executables: %v", err)
+	}
+
```

Comment:
Explicitly add a check for `len(paths) == 0` and show appropriate message that the command is not installed.